src/test/scala/uk/gov/hmrc/test/ui/pages/CommonElements.scala

```diff
@@ -91,6 +91,7 @@ trait CommonElements extends BasePage with Matchers{
     "change pay received"             -> By.cssSelector("#main-content > div > div > dl > div:nth-child(6) > dd.govuk-summary-list__actions.govuk-\\!-width-one-third > a"),
     "change tax taken from pay"       -> By.cssSelector("#main-content > div > div > dl > div:nth-child(7) > dd.govuk-summary-list__actions.govuk-\\!-width-one-third > a"),
     "third remove employment"         -> By.cssSelector("#main-content > div > div > div:nth-child(3) > ul > li:nth-child(3) > span.hmrc-add-to-a-list__remove > a"),
+    "first change employment expenses" -> By.cssSelector("#main-content > div > div > dl > div > dd.govuk-summary-list__actions.govuk-\\!-width-one-third > a"),
```

Comment:
Could probably make use of the "clicks on the change link in position 1" step def instead of adding this?

Reply:
Yes I have used this - never saw this version just the one that took a section and position which I couldn't se

Re-used this one